packages/zowe-explorer-api/src/tree/IZoweTreeNode.ts

```diff
@@ -352,6 +352,8 @@ export interface IZoweUSSTreeNode extends IZoweTreeNode {
      */
     getAttributes(): Types.FileAttributes | PromiseLike<Types.FileAttributes>;
 
+    fetchAttributes(): Types.FileAttributes | PromiseLike<Types.FileAttributes>;
```

Comment:
Sorry I didn't catch this during the last round of review, but if we need to define a new function on this interface, it has to be marked optional to avoid breaking changes for extenders.

Reply:
thanks @traeok good catch! addressed in latest commit